src/orders/Dockerfile

```diff
@@ -3,20 +3,57 @@ ARG APPLICATION_NAME="orders"
 ARG VERSION="0.0.1-SNAPSHOT"
 
 # ------------
-# Stage 1 : Setup the build environment
-FROM gradle:6.5 as buildenv
+# Stage 1 : Setting up the build environment with maven and gradle
+# 
+# 
+FROM maven:3.6-jdk-8 as buildenv
+
+ARG JDBC_DRIVER_VERSION
+ARG JDBC_GROUP_ID="com.oracle.jdbc"
+ARG JDBC_DRIVER_ARTIFACT_ID="ojdbc8"
+ARG GRADLE_VERSION=5.6.1
+
+ENV GRADLE_HOME /opt/gradle
+
+# COPY --from=oracleinstantclientjdbc /usr/lib/oracle/${JDBC_DRIVER_VERSION}/client64/lib/ojdbc8.jar /usr/lib/oracle/${JDBC_DRIVER_VERSION}/client64/lib/ojdbc8.jar
```

Comment:
What is the reason to have commented code on the scripts?

Reply:
I can remove that.